resources/typemappings/Genome.yaml

```diff
@@ -0,0 +1,35 @@
+storage-type: WS
+types:
+    KBaseGenome.Genome:
+        default:
+           -
+               type: Genome
+               version: 1
+           -
+               type: GenomeFeature
+               version: 1
+
+        versions:
```

Comment:
Don't mix production and CI specs in the same mapping - there will have to be one mapping for production and one for CI. This mapping will map ci version 9 to the search version 2